test/Tester/OrleansLoggingTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Orleans.Extensions.Logging;
+using Orleans.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Tester
+{
+    [TestCategory("BVT"), TestCategory("OrleansLogging")]
+    public class OrleansLoggingTests
+    {
+        [Fact]
+        [Obsolete]
```

Comment:
Why are these tests marked obsolete?

Reply:
to avoid compile warnings

Reply:
Can't we pragma disable the warnings for the logging tests instead of putting on ```Obsolete``` on the test methods?